data/resume/jbatorski.json

```diff
@@ -0,0 +1,182 @@
+{
+  "$schema": "https://raw.githubusercontent.com/jsonresume/resume-schema/v1.0.0/schema.json",
+  "basics": {
+    "name": "Jerry Batorski",
+    "label": "Frontend Software Engineer",
+    "email": "jerry.batorski@gmail.com",
+    "url": "https://jarekb84.github.io/me/",
+    "summaryOfQualifications": [
+      "11+ YOE as a software engineer, with a focus on the frontend",
+      "9+ YOE breaking down projects, communicating with stakeholders, delivering business value, and adapting to change",
+      "3+ YOE leading a frontend focused squad of software engineers"
+    ]
+  },
+  "work": [
+    {
+      "name": "Grubhub",
+      "location": "Chicago, IL",
+      "url": "https://www.grubhub.com/",
+      "startDate": "2016-11-01",
+      "skills": [
+        "JavaScript",
+        "React",
+        "Redux",
+        "SCSS",
+        "Jasmine/Karma",
+        "TestCafe",
+        "Jest",
+        "RTL",
+        "TypeScript",
+        "SCRUM",
+        "JIRA",
+        "Git"
+      ],
+      "positions": [
```

Comment:
Positions and skills aren't standard properties on the jsonresume schema, see this [comment](https://github.com/jarekb84/me/pull/3/files#r821286995) for reasoning.